index.html

```diff
@@ -170,6 +186,9 @@ <h2 class="m-0 text-lg pt-4">Examples</h2>
 				<a href="https://shield.deno.dev/x/postgres" target="_blank">https://shield.deno.dev/x/postgres (module under 1.0)</a>
 				<img src="/x/postgres" alt="postgres deno module version"/>
 
+				<a href="https://shield.deno.dev/std/^1.35" target="_blank">https://shield.deno.dev/std/0.210.0 (std version)</a>
```

Comment:
Thanks for the PR. Wrong URL here.
Also please use snake case for all variables and function names.

Reply:
alright. would the `snake_case` rule apply to `index.html`? or whole project-wide? i think `mod.ts` used `pascalCase`.

Reply:
Snake case applies to the whole project.

Reply:
> Snake case applies to the whole project.

added it to `README.md` as it wasn't stated.